views/containers/GameLoader.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import Game from '../components/Game';
+import LoadingScreen from '../components/LoadingScreen';
+import * as BrowserActions from '../actions/Browser';
+
+class GameLoaderComponent extends React.Component {
+  constructor() {
+    super();
+    this.didLoad = false;
+  }
+
+  componentDidMount() {
+    setTimeout(()=>{
```

Comment:
We need to use `redux-thunk` to handle async operations from actions, since this should be in an action because you are coupling the component with the API. Also, as a future reference, every time you create a `timeout` it must be cleared, in this case it should be cleared on `componentWillUnmount`.

```javascript

componentDidMount() {
   this.timeout = setTimeout(...);
}

componentWillUnmount() {
  clearTimeout(this.timeout);
}
```